CMakeLists.txt

```diff
@@ -8,11 +8,6 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/")
 
-if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
```

Comment:
I suggest to remove this: The cmake file shouldn't impose any (non-required) build setting.

Reply:
This check was added throughout MoveIt packages to avoid bad first-time experiences for new users building from source. Solving planning problems can be apparently an order of magnitude slower when building with a standard pipeline and cmake's default.

If this is still the case (I don't see why the situation would have changed) I would vote to keep it.

Reply:
Ok.

Reply:
The only problem I see is that `colcon` prints all messages as warnings (actually, printing to `stderr`) which are treated as errors by Travis. We would have to add `-DCMAKE_BUILD_TYPE=Release` to each `colcon build` command

Reply:
Done in https://github.com/ros-planning/moveit_ci/pull/95